android/src/main/java/com/horcrux/svg/PathParser.java

```diff
@@ -39,6 +39,10 @@ Path parse(String data) {
         while (i < l) {
             skip_spaces();
 
+            if (i >= l) {
+                break
```

Comment:
Was it complaining about the missing ";" when u fixed this? It is complaining on mine D: